workflow/rules/common.smk

```diff
@@ -45,9 +45,18 @@ wildcard_constraints:
     unit="N|T|R",
 
 
+ruleorder: sort_vcf > bgzip_vcf
+
+
 def compile_output_list(wildcards):
-    return [
-        "filtering/add_multi_snv_in_codon/%s_%s.codon_snvs.sorted.vcf.gz" % (sample, t)
+    output_files = [
```

Comment:
I think this should be done programmatically and not using append because you are using the same wildcards.

Reply:
done